### DISCUSSION TITLE
Esp Muse Luxe not showing

### DISCUSSION THREAD

**Discussion by orrionis** [music-assistant/support#1880] [Q&A] [answered]
Hello everybody

I have bought a ESP Muse Luxe from raspyaudio.

i see lot of  video of guys who add and play music on via music assistant.

I flash the speaker in media player . i add to home assistant and it was see as a player, i can send music on. all is funny 

but music assitant don't see . all video are old than one year and with the first version of MA.

Do you have an idea why it don't see my speaker ?

Thanks

**Comment by OzGav** [comment, marked as answer]
Only ESP players running squeezelite will be seen natively by MA. However, as of beta 93 you should be able to play to the HA entitiy.

**Reply by uSlackr** [reply]
I have this same device.  It is seen as a HA media  player and I added it to MA, but it will not send music to it.  I increased the log level to verbose but see no entries for the player attempt

**Reply by uSlackr** [reply]
I fixed this by opening the ESPHome integration in HA, locating the device and clicking configure and enabling the device to "Allow the device to perform Home Assistant actions"

**Reply by jd1** [reply]
Is it stable? I'm using the ESP Muse Luxe with squeezelite. It is recognized and playing music works, but after some time it stops playing.
